### PR TITLE
feat!: Disallow blue Spotlight and make purple the default

### PR DESCRIPTION
**Pull request by VincentSmedinga**
# Describe the pull request

Thank you for contributing to the project!
Please use this template to help us handle your PR smoothly.

## What

This removes the blue option for the colour of a Spotlight.

## Why

We don’t want the appearance of Spotlight to interfere with that of Footer.

## How

- Changed the default to purple.
- Applied the default to the main component class.

## Checklist

Before submitting your pull request, please ensure you have done the following. Check each checkmark if you have done so or if it wasn't necessary:

- [x] Add or update unit tests
- [x] Add or update documentation
- [x] Add or update stories
- [x] Add or update exports in index.\* files
- [x] Start the PR title with a Conventional Commit prefix, [as explained here](https://github.com/Amsterdam/design-system/blob/main/documentation/publishing.md?plain=1#L11).

## Additional notes

- The tests will succeed after completing #1797 and merging develop into this branch.

### CODE REVIEW

File: packages/css/src/components/spotlight/README.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ Emphasizes a section on a page through a distinctive background colour.
 ## Guidelines
 
 - Display the Spotlight at the entire width of the [Screen](/docs/components-layout-screen--docs); do not position it on the [Grid](/docs/components-layout-grid--docs).
+- Do not use other background colours than the options below.
+  The blue appearance of Footer must remain unique – this is why that colour is not an option for Spotlight.
+  Red, black and grey aren’t options either.
```
Comment (alimpens) [inline packages/css/src/components/spotlight/README.md:12]: Do we need to describe this? If we just don't offer it as an option, we already communicate this, no?

File: packages/react/src/Spotlight/Spotlight.tsx [hunk 2]
```diff
@@ -6,9 +6,10 @@
 import clsx from 'clsx'
 import { forwardRef, HTMLAttributes, PropsWithChildren } from 'react'
 
-export const spotlightColors = ['azure', 'blue', 'green', 'lime', 'magenta', 'orange', 'purple', 'yellow'] as const
+const defaultColor = 'purple'
```
Comment (alimpens) [inline packages/react/src/Spotlight/Spotlight.tsx:9]: If this is an unchanging value (as in, a true constant), the convention is to write it in all caps.

File: packages/react/src/Spotlight/Spotlight.tsx [hunk 3]
```diff
@@ -18,8 +19,12 @@ export type SpotlightProps = {
 } & PropsWithChildren<HTMLAttributes<HTMLElement>>
 
 export const Spotlight = forwardRef(
-  ({ children, className, as: Tag = 'div', color = 'blue', ...restProps }: SpotlightProps, ref: any) => (
-    <Tag {...restProps} ref={ref} className={clsx('ams-spotlight', `ams-spotlight--${color}`, className)}>
+  ({ as: Tag = 'div', children, className, color = defaultColor, ...restProps }: SpotlightProps, ref: any) => (
+    <Tag
+      {...restProps}
+      ref={ref}
+      className={clsx('ams-spotlight', { [`ams-spotlight--${color}`]: color !== defaultColor }, className)}
```
Comment (alimpens) [inline packages/react/src/Spotlight/Spotlight.tsx:26]: This condition is a little hard to read imo. How about:

```suggestion
      className={clsx('ams-spotlight', { color !== defaultColor && `ams-spotlight--${color}` }, className)}
```

File: packages/react/src/Spotlight/Spotlight.tsx (hunk 2 above)
Comment (VincentSmedinga) [reply packages/react/src/Spotlight/Spotlight.tsx:9]: We decided to postpone this possible change.

File: packages/css/src/components/spotlight/README.md (hunk 1 above)
Comment (VincentSmedinga) [reply packages/css/src/components/spotlight/README.md]: Yes, removed.

**Review by alimpens — APPROVED** [review]

**Pull request merged**